dev-util/SPIRV-Headers/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>x11@gentoo.org</email>
```

Comment:
Did @gentoo/x11 approve this? You don't 'give' maintenance over to random people in Gentoo.

Reply:
yes, i am working with mattst88 from the X11 team on this work